hal-gen/src/main.rs

```diff
@@ -32,21 +32,25 @@ impl bindgen::callbacks::ParseCallbacks for BindgenCallbacks {
             Some("tResourceType") => {
                 Some(original_variant_name["kResourceType_".len()..].to_owned())
             }
-            Some(enum_name) => {
-                if original_variant_name.starts_with(enum_name) {
-                    Some(original_variant_name[enum_name.len() + 1..].to_owned())
-                } else {
-                    None
-                }
+            Some(enum_name) if original_variant_name.starts_with(enum_name) => {
+                Some(original_variant_name[enum_name.len() + 1..].to_owned())
             }
-            None => None,
+            _ => None,
+        }
+    }
+
+    fn will_parse_macro(&self, name: &str) -> bindgen::callbacks::MacroParsingBehavior {
```

Comment:
In the bindings on my machine the errors messages don't show for me anyway.

Reply:
Wait, I see now. This is needed because of the addition of `_` to the regex with `\w`. Nevermind.